src/pages/Detail/DetailMain/DetailMain.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ProductImg from './ProductImg';
+import DetailDesc from './DetailDesc';
+import './DetailMain.scss';
+
+const DetailMain = ({ productBox }) => {
+  return (
+    <div className="detailMain">
+      <div className="productImg-container">
+        {productBox[0] &&
+          productBox[0].img.map(product => {
```

Comment:
```suggestion
        {productBox[0] &&
          productBox[0].img.map(({id, src}) => {
```
매개변수자리에서 바로 구조분해할당 가능합니다!